data/presets/shop/deli.json

```diff
@@ -5,12 +5,19 @@
         "area"
     ],
     "terms": [
+        "deli",
+        "delicatessen",
         "lunch",
         "meat",
         "sandwich"
     ],
     "tags": {
         "shop": "deli"
     },
-    "name": "Deli"
+    "name": "Specialty Food Store",
```

Comment:
Is [<img src="https://raw.githubusercontent.com/ideditor/temaki/58eedd7e2a548e2c3319cdba30911f221912725e/icons/meat.svg" height="24" align="top" alt="temaki-meat">](https://github.com/ideditor/temaki/blob/58eedd7e2a548e2c3319cdba30911f221912725e/icons/meat.svg) a juicy steak still the best icon for this preset given how it’s currently defined? I think it’s slightly more intuitive than the [<img src="https://wiki.openstreetmap.org/w/images/3/3b/Deli-14.svg" height="24" align="top" alt="">](https://wiki.openstreetmap.org/wiki/File:Deli-14.svg) lotion bottle that openstreetmap-carto uses, but it doesn’t really distinguish the preset from, say, a butcher. Maybe a sausage link?

Reply:
fair enough, I found a similar icon on font awesome:
https://fontawesome.com/icons/jar?s=solid

Reply:
I don't think a sausage would be representative (more) for a delicatessen shop. Oftentimes, delicatessen are sold as non-perishables, e.g. pickled, canned, cured or some kind of jam. So, a jar is fine IMO. But if you find sausages to be better, I guess you could create a separate PR. This PR is mostly about the aliases.

Reply:
Yes, understood. Sorry for scope-creeping. If non-perishables are the main fare of a delicatessen, then definition and infobox image [on this page](https://wiki.openstreetmap.org/wiki/Tag:shop%3Ddeli) could be improved to clarify that focus. I don’t have a problem with representing this preset with a jar, except that it’s difficult to make a jar of food distinguishable from anything else. But this can be improved separately.

Reply:
No, not the main fare. Just common.